.travis.yml

```diff
@@ -2,8 +2,5 @@ language: python
 python:
   - "2.6"
```

Comment:
I'd suggest testing 2.7 as well since that's what EL7 ships. IMHO way more important than 2.6 from EL6 ;)

Reply:
I found [some ansible documentation](http://docs.ansible.com/ansible/latest/dev_guide/developing_python3.html#minimum-version-of-python-3-x-and-python-2-x) suggesting that we should use Python 2.6 and 3.5